v2/src/ffi/py/value.rs

```diff
@@ -151,7 +137,7 @@ impl Message {
     pub fn __repr__(&self) -> String {
         format!(
             "Message(role={}, id={:?}, thinking={:?}, contents=[{}], tool_calls=[{}], signature={:?})",
```

Comment:
```suggestion
            "Message(role=\"{}\", id={:?}, thinking={:?}, contents=[{}], tool_calls=[{}], signature={:?})",
```

I just saw `Message` is printed as `Message(role=system, ...)`, but it's better to be `Message(role="system", ...)`.
So suggesting change above. (same for L265 too!)